openapi_tester/loaders.py

```diff
@@ -128,34 +130,54 @@ def resolve_path(self, endpoint_path: str) -> tuple:
         parsed_path = url_object.path if url_object.path.endswith("/") else url_object.path + "/"
         if not parsed_path.startswith("/"):
             parsed_path = "/" + parsed_path
+        for key, value in self.field_key_map.items():
```

Comment:
So this enables us to optionally override, for things like the i18n feature, while coerced DRF paths should be automatically replaced?